CMakeLists.txt

```diff
@@ -848,8 +848,8 @@ if(CC_USE_AUDIO)
         ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/tinysndfile.h
         ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/Track.cpp
         ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/Track.h
-        ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/UrlAudioPlayer.cpp
-        ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/android/UrlAudioPlayer.h
+        ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/openharmony/UrlAudioPlayer.cpp
+        ${CMAKE_CURRENT_LIST_DIR}/cocos/audio/openharmony/UrlAudioPlayer.h
```

Comment:
Can we put together files from the same platform?

Reply:
done